src/index.js

```diff
@@ -94,7 +92,7 @@ class IPFSPubsub {
     let content
 
     // Get the topic
-    const topicId = message.topic
+    const topicId = message.topicIDs[0]
```

Comment:
Noting here that this is changed due to the pubsub message data structure change.